src/entries/popup/components/ExternalImage/ExternalImage.tsx

```diff
@@ -137,9 +137,9 @@ const ExternalImage = (props: ExternalImageProps) => {
           ? { borderRadius: props.borderRadius }
           : {}),
         boxShadow: isLoading || error ? undefined : props.boxShadow,
-        height: props.height,
         overflow: error ? 'visible' : 'clip',
-        width: props.width,
+        height: props.height && (Number(props.height) ?? props.height),
+        width: props.width && (Number(props.width) ?? props.width),
```

Comment:
i'm a bit confused, why is this?

Reply:
when passing `height="20"` it didn't work but `height={20}` did, I think it expect to have a unit like `height="20px"` when it's a string (?)